src/Microsoft.PowerShell.Commands.Utility/commands/utility/Var.cs

```diff
@@ -339,7 +339,7 @@ protected override void ProcessRecord()
                     matchFound = true;
                     if (_valueOnly)
                     {
-                        WriteObject(matchingVariable.Value);
+                        WriteObject(matchingVariable.Value, true);
```

Comment:
I think this is more clear:
`WriteObject(matchingVariable.Value, enumerateCollection: true);`

Reply:
Yeah, I'll go ahead and do that instead. Thanks!